website/docs/user_guide/environments.md

```diff
@@ -56,6 +56,14 @@ In order for the SDK to download the feature toggle configuration for the correc
 
 ![An API key creation form. The form's fields are "username", "token type", "project", and, crucially, "environment". The development environment is selected.](/img/environments_api_keys.png 'Create Environment specific API Keys')
 
+## Cloning environments
+
+If you have a lot of feature toggles, it can be tedious to configure the same activation strategies for each environment. To make this easier, you can clone an environment. This will copy all feature toggle configurations and activation strategies from the source environment to the target environment.
+
+This can also prove to be a safe way to experiment with activation strategies without affecting production-facing environments. You can clone the production environment, and then test out the new activation strategies in the cloned environment. If everything works as expected, you can then apply the same activation strategies to the production environment.
+
+When cloning an environment, you can choose several options to customize your new environment: name, type of environment, which projects to clone the environment configuration in, and whether you wish to keep the same user permissions. You'll also be provided the choice of generating a new API key bound to the newly cloned environment right away, or generating one yourself at a later stage.
```

Comment:
I'm not sure I understand: the first paragraph reads as if you can choose to copy strategies from an existing environment into another existing environment (e.g. from "dev" to "production"). I might have misunderstood, but I thought this feature only allowed you to create **new** environments that contained the same configuration as the source?

If that's correct, maybe we can be reword this section into something like this:

```suggestion
Unleash environments can be cloned. Cloning an environment creates a **new** environment based on the selected source environment. When cloning an environment, you select any number of projects whose feature toggle configurations will also be cloned. These projects will have identical configurations for the source and target environments. However, the environments can then be configured independently and will not stay in sync with each other.

When cloning an environment, you must give the new environment
- a name
- an environment type
- a list of projects to clone configurations in

You must also decide whether you want to clone the user permissions to the new environment or ... [what do you do here? what's the option?]

Once created, the new environment works just as any other environment.
```

I like the use case suggestions, but I feel like they might be better suited in a "### Use cases" subsection? Wdyt?